Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -16997,6 +17010,21 @@ end
 				{
 					Include = "^E_META_.*",
 				},
+				EffectID = 
```

Comment:
Exactly, and the name of the enum is `SmokeDirection`, not `EffectID`. Not to mention that having a duplicate entry in the description table silently hides the previous same-name entries.